src/cow-react/utils/fractionToString.ts

```diff
@@ -0,0 +1,18 @@
+import { Fraction } from '@uniswap/sdk-core'
+
+export function fractionToString(f: Fraction | null | undefined): string {
```

Comment:
I was unsure about that this function would do by reading the name. 
I mean, I could expect that its converting a Fraction into string, but which format? 
- string decimal? `0.2`
- string representing the fraction `1/5`
- JSON string: `{ numerator: 1, denominator: 5 }`

I would think here what you are doing the 3rd one, which I would call it to `serialize` it, so you can `deserialize` later?

Reply:
Good point! Refactored it to `FractionUtils`